__tests__/app.test.js

```diff
@@ -12,4 +28,13 @@ describe("GET /api", () => {
         expect(endpoints).toEqual(endpointsJson);
       });
   });
+  // test("404: Responds with message that URL is not found", () => {
+  //   return request(app)
+  //     .get("/app")
+  //     .expect(404)
+  //     .then((res) => {
+  //       console.log(res.body);
+  //       // expect(msg).toBe("Not found");
+  //     });
+  // });
```

Comment:
When you get this to work I would put it in it's own describe block - because it is on a different path to /api. 

Also, you only need to test for this once, so you don't need to do this for every path. I would make your get path more clear like - /notaroute maybe